src/mlpack/methods/kde/kde_main.cpp

```diff
@@ -134,6 +193,22 @@ static void mlpackMain()
   ReportIgnoredParam({{ "input_model", true }}, "kernel");
   ReportIgnoredParam({{ "input_model", true }}, "rel_error");
   ReportIgnoredParam({{ "input_model", true }}, "abs_error");
+  ReportIgnoredParam({{ "input_model", true }}, "monte_carlo");
+  ReportIgnoredParam({{ "input_model", true }}, "mc_probability");
+  ReportIgnoredParam({{ "input_model", true }}, "initial_sample_size");
+  ReportIgnoredParam({{ "input_model", true }}, "mc_entry_coef");
+  ReportIgnoredParam({{ "input_model", true }}, "mc_break_coef");
```

Comment:
Hmmm, should we have the user always specify `--monte_carlo` if they want to do Monte Carlo sampling for search?  Imagine this possibility: a user builds a KDE model with `--monte_carlo` specified, but now they want to run a search with it turned *off*.  I think the design now doesn't support that, but do you think it would be reasonable to change it so that that could be done?  Technically the model only really consists of the tree and its parameters, so at query time it should be feasible to change whether Monte Carlo sampling is done or not.

(That should also be true for `rel_error` and `abs_error`, I guess.  I suppose I missed that when I reviewed the original KDE PR. :))

Reply:
About the first monte_carlo=false by default question: Yes, they need to specify they want to do Monte Carlo sampling. I thought this would be a good default since it produces a non-deterministic output and maybe it's a good tool for users with more expertise. As I don't use KDE everyday I don't know that well if it would be good to use Monte Carlo by default, so if you think it's better then I can quickly change it :)

About the rest of questions. Modifiers are now implemented and users can change parameters for loaded models :)

Reply:
Before merging: 
1. What do you think about `--monte_carlo` being deactivated by default?
2. Do you think the rest of defaults are ok? So far, they seem to work well.

Reply:
I think deactivated by default probably makes the most sense (although I could be convinced either way), and the rest of the defaults seem good to me. :+1: